src/main/java/com/mongodb/hibernate/service/spi/MongoConfigurationContributor.java

```diff
@@ -54,12 +54,13 @@
 public interface MongoConfigurationContributor extends Service {
 
     /**
-     * Configures the MongoDB Extension for Hibernate ORM. This method is called once per instance of
-     * {@link StandardServiceRegistry} that has this {@link MongoConfigurationContributor}
-     * {@linkplain StandardServiceRegistryBuilder#addService(Class, Service) added}.
+     * Configures the MongoDB Extension for Hibernate ORM.
      *
      * @param configurator The {@link MongoConfigurator} pre-configured with {@linkplain Configurable#configure(Map)
      *     configuration properties}.
+     * @mongoCme This method is called once per instance of {@link StandardServiceRegistry} that has this
```

Comment:
Given that `MongoConfigurationContributor` is part of our API, you can see how the new tag looks in the generated docs.